help/user-guide/isa-json-compliant-experiment.md

```diff
@@ -49,69 +54,33 @@ The outputs of an Assay can only be used as inputs by the next Assay in the same
 * data files (e.g. files containing measurements, rawdata.fastq, processed data, reads.counts.txt).
 
 
-# Experiment Sample Templates
-
-Experiment Sample Templates act as blueprints to create Sample Types within ISA Studies and ISA Assays. The same Experiment Sample Template can be applied multiple times to create Sample Types in different ISA Studies or ISA Assays.
+## Experiment Sample Templates
```

Comment:
@somathias why shouldn't this be main title (only 1 #)?

Reply:
Personally, I prefer if "ISA-JSON compliant investigation" is the main title and all other information in the same file is organised under it at suitable levels. (Formulated in a different way: why did you have multiple main titles in the same .md file?)